pavelib/bok_choy.py

```diff
@@ -130,3 +165,25 @@ def bokchoy_coverage():
     sh("coverage html --rcfile={}".format(coveragerc))
     sh("coverage xml --rcfile={}".format(coveragerc))
     sh("coverage report --rcfile={}".format(coveragerc))
+
+
+@task
+def bokchoy_coverage():
```

Comment:
I'm not sure if this would need to be called as part of our nightly 'bok choy coverage' job. Could you confirm whether or not any changes are needed there?